src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/RemoveProviderModal.jsx

```diff
@@ -18,8 +18,8 @@ export default function RemoveProviderModal({ onClose, provider, address }) {
         {distanceBetween(
```

Comment:
Currently, the modal shows "NaN miles" (see the captured screenshot on this PR). I'm wondering if it's better to set the condition to check if address exist then call out function `distanceBetween` otherwise display miles that you can pass into the function by the way of ProviderSelect.jsx

      {showRemoveProviderModal && (
        <RemoveProviderModal
          miles={formData[sortMethod]}
          provider={formData}
          address={address}
          onClose={response => {
            setShowRemoveProviderModal(false);
            if (response === true) {
              setCheckedProvider(false);
              onChange({});
            }
          }}
        />
      )}
Let me know if this makes sense. We can certainly do a slack call.

Reply:
I would also ask @laurenernest or Peter what should display when there is no address on file. We already display a notification stating that some of the features might not work when there is no address at the start of scheduling an appointment. This seems like a candidate. I would probably modify the distanceBetween function since it currently assumes all the parameters are not null.

Reply:
Wow this was an outright "swing and a miss".....thanks for the catch......@laurenernest I will research how the 1,400 mile info was calculated while I wait to hear back on UX.

Reply:
okay @jenniemc ...thank you for the help and suggestions yesterday. I implemented the fix you recommended and it should be ready for review

Reply:
@jenniemc ....Thank you for all the help and suggestions yesterday. I have pushed up the changes we talked about so, this PR should be good for another review.